deck.gl__core/index.d.ts

```diff
@@ -1280,8 +1280,6 @@ declare module "@deck.gl/core/lib/composite-layer" {
 		get isComposite(): boolean;
 		get isLoaded(): any;
 		getSubLayers(): any;
-		initializeState(params?: any): void;
```

Comment:
`CompositeLayer` extends `Layer`, therefore these types should not be required. I've tested this locally, but please let me know if I've overlooked something.

Reply:
as long as it builds. Can you run `npm test` locally without error?

Reply:
Had a few issues with tests in the end due to a recent release of `luma.gl` including types which were conflicting with those in this library. As a result, I've had to specify versions of luma that do not include said types. This is a dev dependency so it won't have any impact on end users, however, I've raised an issue (#137) to look at resolving this properly.